code/src/components/Form.js

```diff
@@ -0,0 +1,296 @@
+							key={'button-next'}
+							onClick={() => nextPage(traitsCheckboxes)}
+							text='Next'
+						/>
+					</div>
+				</div>
+			)}
+			{page === 5 && (
+				<div className='current-page question'>
+					<div>
+						<h2>{questions.backTogetherQuestion}</h2>
+						{questions.backTogetherAlts.map((alt) =>
+							<div key={alt}>
+								<RadioButton
+									alt={alt}
+									onChange={handleGetBackTogether}
+									checked={getBackTogether === alt}
+								/>
+							</div>
+						)}
+					</div>
+					{(getBackTogether === "Never in a million years!" || getBackTogether === "Nah.") && (
+						<div>
+							<h2>{questions.dateFriendQuestion}</h2>
+							{questions.backTogetherAlts.map((alt, index) =>
+								<div key={alt+index}>
+									<RadioButton
+										alt={alt}
+										onChange={handleDateFriend}
+										checked={dateFriend === alt}
+									/>
+								</div>
+							)}
+						</div>
+					)}
+					{(dateFriend === "Absolutely!" || dateFriend === "Maybe?") && (
+						<div className='text-input-container'>
+							<Input 
+								label={<h2>{questions.friendNameQuestion}</h2>}
+								onChange={handleFriendContactDetails}
+								value={friendContactDetails}
+								placeholder={'Name, number'}
+							/>
+						</div>
+					)}
+					<div>
+						<h3 className='error'>{error}</h3>
+						<Button 
+							onClick={backPage}
+							text='Back'
+						/>
+						<Button 
+							onClick={() => nextPage(getBackTogether)}
+							text='Send'
+						/>
+					</div>
+				</div>
+			)}
+			{page === 6 && (
+				<div className='current-page summary-page'>
+					<h2>Dear {userName}</h2>
+
+					<p>I'm sorry {breakUpper()[0]} broke up with {breakUpper()[1]}...</p>
+
+					<p> We were together for {shipDuration} and I really cherish the time we had together!</p>
+
+					<p>You say the reason we broke up was, and I quote, '{breakupReason}', but we both know the real reason.</p>
+
+					{traitsCheckboxes.length > 0
+						? traitsCheckboxes.includes("All of the above")
+							? (<p>I know you think I have all the worst qualities a human can have and you are probably right... I'll try to be 		better!</p>)
+							: (<p>I know you think I'm {traitsCheckboxes.join(', ')} and you are probably right... I'll try to be better!</p>)
+						: breakUpper()[0] === "I" 
+							? <p>I know you think I'm flawless but I'm stupid for breaking up with you!</p>
+							: <p>Why did you break up with me if I'm so flawless?!</p>
+					}
+
+					{reconciliation()
+						? <p>I also believe we should give it another go!</p>
+						: (dateFriend === "Absolutely!" || dateFriend === "Maybe?")
+							? {friendContactDetails}
+								? <p>Thank you for being so open minded as to let me date your friend! I'll definitely contact them!</p>
+								: <p>Thanks, but was it asking too much to give me their number?</p>
+							: <p>I can accept that you don't want to give me another chance, but why can't I have your friend's number?</p>
+					}

```

Comment:
This is also just magnificent - what a great summary. The ternary operators and functions.... I honestly am having trouble wrapping my mind around lines 275 - 291. So impressed with everything that's going on behind the scenes here!